### DISCUSSION TITLE
My school doesn’t give student edu email but apply tell my I have to

### DISCUSSION THREAD

**Discussion by cdwcgt** [community/community#69874] [GitHub Education] [answered]
### Hi everyone,

As above, I just asked my college counselor, he said that my school doesn’t give edu email to student.
So I need to apply education pack by using personal email.
But my school is in list and when I select it, apply form require a edu email(continue button disabled)

I can provide other evidence to prove that I am a student of this school, which like chsi verifiy document that can be used for jetbrain’s education discount certification(for student in china)

so is there any method to apply?

**Comment by muhammedfurkan** [comment, marked as answer]
open a ticket about this , add student document or someting like that. github staff will be in touch with you.
